developments_build/version.env

```diff
@@ -1,5 +1,6 @@
-CAPTURE_DATE=2020-06-30
-CAPTURE_DATE_PREV=2019-12-31
-DOB_DATA_DATE=2020/07/01
-VERSION=20Q2
-VERSION_PREV=19Q4
\ No newline at end of file
+CAPTURE_DATE=2021-01-03
+CAPTURE_DATE_PREV=2020-06-30
+DOB_DATA_DATE=2020/01/01
```

Comment:
should the capture date be 2021-01-01? and dob date as 2021/01/04?

Reply:
![image](https://user-images.githubusercontent.com/13207770/103773512-2b7d1200-4ff9-11eb-9cc9-afb405b4c4ae.png)